src/test/java/wromaciej/hvac_sim/thermo/fluids/FluidFactoryTest.java

```diff
@@ -43,11 +42,14 @@ public void shouldReturnAbout4200JperkgKForStandardWaterHeatCapacity() {
     @Test
     public void shouldReturn74gPerKgForStandardAir() {
         //GIVEN
+        FluidData fluidData = new FluidData();
+        fluidData.loadLibrary();
+        FluidFactory fluidFactory= new FluidFactory(fluidData);
```

Comment:
I think that `FluidData` and `FluidFactory` are stateless, so you could create them once at the beginning of test as test class field, and reuse between tests